src/main/java/org/tron/core/config/args/Args.java

```diff
@@ -74,6 +72,9 @@
   @Parameter(names = {"-p", "--private-key"}, description = "private-key")
   private String privateKey = "";
 
+  @Parameter(names = {"--password"}, description = "password")
+  private String password = "";
```

Comment:
![Vulnerability](https://sonarsource.github.io/sonar-github/vulnerability.png 'Vulnerability') Vulnerability: 'password' detected in this expression, review this potentially hardcoded credential. ([squid:S2068](https://sonarcloud.io/organizations/tron-zhaohong/rules#rule_key=squid%3AS2068))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=java-tron&pullRequest=950&issues=AWPTFGGfA0k93xG_IQNZ&open=AWPTFGGfA0k93xG_IQNZ)

[ISSUE_KEY]: AWPTFGGfA0k93xG_IQNZ